ztp/gitops-subscriptions/argocd/example/image-based-upgrades/README.md

```diff
@@ -0,0 +1,91 @@
+
+```plaintext
+├── source-crs/
+│   ├── ibu/
+│   │    ├── PlatformBackupRestore.yaml
+├── ...
+├── custom-oadp-workload-crs.yaml
+├── example-oadp-policy.yaml
+├── kustomization.yaml
+```
+
+Note that [`source-crs/ibu/PlatformBackupRestore.yaml`](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/ibu/PlatformBackupRestore.yaml) is provided in the ZTP image, however, it is important to ensure that the `kustomization.yaml` file is located in the same directory structure shown above in order to reference the `PlatformBackupRestore.yaml` manifest.
+
+### Generating the ConfigMap and Policy
+
+Use the [`configMapGenerator`](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#configmapgenerator) provided by the Kustomize tool to generate a ConfigMap encapsulating the OADP CRs for IBU. The OpenShift platform-related CRs are available [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/ibu/PlatformBackupRestore.yaml).
+
+Additional workload OADP CRs may be included in the ConfigMap generation process as shown in the example below:
+
+```yaml
+configMapGenerator:
+- files:
+  - source-crs/ibu/PlatformBackupRestore.yaml
+  - custom-oadp-workload-crs.yaml
+  name: oadp-cm
+  namespace: ztp-common
+
+generatorOptions:
+  disableNameSuffixHash: true
+```
+
+Ensure that the `custom-oadp-workload-crs.yaml` file includes a one-to-one mapping of OADP backup and restore CRs. It's important to note that these CRs can be stored either in separate YAML manifests or consolidated within a single YAML file (as shown below), with each CR section separated by the `---` directive.
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: foobar-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - foobar
+  includedNamespaceScopedResources:
+  - secrets
+  - deployments
+  - statefulsets
+  excludedClusterScopedResources:
+  - persistentVolumes
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: foobar-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
+spec:
+  backupName:
+    foobar-app
+```
+
+> [!IMPORTANT]
+> Don't forget to create and include the policy [example-oadp-policy.yaml](./example-oadp-policy.yaml) under the `resources` object in the `kustomization.yaml` file.
+> The policy uses hub-side templating and requires Advanced Cluster Management (ACM) 2.9 and later versions.
+
+
+### Enforcing the Policy
+
+To enforce the policy and distribute the ConfigMap to the applicable managed clusters, create a ClusterGroupUpgrade (CGU) CR referencing the aforementioned policy.
+
+For more detailed information on using the Life Cycle Agent (LCA) operator, refer to the [docs](https://github.com/openshift-kni/lifecycle-agent/tree/main/docs).
```

Comment:
There is a procedural element that needs to be discussed somewhere -- specifically this configmap must be created during which phase of the upgrade process? ie how does this ConfigMap get created "in time" for use by LCA? It's fine if that is covered elsewhere, but this section touches on rolling out to clusters.

Reply:
I have updated this section accordingly. There will be an official docs that will detail the procedure (which can be included once available).